modules/electrophysiology_browser/jsx/react-series-data-viewer/src/series/components/SeriesRenderer.js

```diff
@@ -0,0 +1,722 @@
+                        />
+                      </div>
+                    </div>
+                  </div>
+                </div>
+              </div>
+              <div className='row'>
+                <div
+                  className='col-xs-1'
+                  style={{
+                    display: 'flex',
+                    flexDirection: 'column',
+                    alignItems: 'center',
+                    height: viewerHeight,
+                  }}
+                >
+                  {filteredChannels.map((channel) => (
+                    <div
+                      key={channel.index}
+                      style={{
+                        display: 'flex',
+                        height: 1 / MAX_CHANNELS * 100 + '%',
+                        alignItems: 'center',
+                      }}
+                    >
+                      {channelMetadata[channel.index] &&
+                      channelMetadata[channel.index].name}
+                    </div>
+                  ))}
+                </div>
+                <div
+                  className='col-xs-11'
+                  onMouseLeave={() => setCursor(null)}
+                >
+                  <div style={{position: 'relative'}}>
+                    {cursor && (
+                      <SeriesCursor
+                        cursor={cursor}
+                        channels={channels}
+                        interval={interval}
+                      />
+                    )}
+                    <div style={{height: viewerHeight}} ref={getBounds}>
+                      <ResponsiveViewer
+                        transparent={true}
+                        mouseMove={R.compose(setCursor, R.nth(0))}
+                        mouseDown={(v) => {
+                          document.addEventListener('mousemove', onMouseMove);
+                          document.addEventListener('mouseup', onMouseUp);
+                          R.compose(dragStart, R.nth(0))(v);
+                        }}
+                      >
+                        <EpochsLayer/>
+                        <ChannelsLayer
+                          viewerWidth={0}
+                        />
+                        <XAxisLayer
+                          viewerWidth={0}
+                          viewerHeight={0}
+                          interval={interval}
+                        />
+                        <ChannelAxesLayer
+                          viewerWidth={0}
+                          viewerHeight={0}
+                        />
+                      </ResponsiveViewer>
+                    </div>
+                  </div>
+                </div>
+              </div>
+              <div
+                className='row'
+                style={{
+                  marginTop: '25px',
+                  marginBottom: '25px',
+                }}
+              >
+                <div className='col-xs-offset-1 col-xs-11'>
+                  {/*
+                  <button
```

Comment:
@laemtl should this commented out code be kept in here?

Reply:
It's a temporary hack, since the feature is not ready yet, but yeah, it's cleaner to remove it for now.